src/components/vuestic-components/va-radio/VaRadio.vue

```diff
@@ -0,0 +1,227 @@
+<template>
+  <label
+    class="va-radio"
+    :class="computedClass"
+  >
+    <input
+      class="va-radio__input"
+      type="radio"
+      :checked="isActive"
+      :disabled="disabled"
+      :name="name"
+      @change="onClick"
+      @focus="onFocus"
+    />
+
+    <span class="va-radio__icon" :style="iconComputedStyles">
+      <span class="va-radio__icon__background" :style="iconBackgroundComputedStyles"></span>
+      <span class="va-radio__icon__dot" :style="iconDotComputedStyles"></span>
+    </span>
+
+    <span class="va-radio__text">
+      <slot>
+        {{ computedLabel }}
+      </slot>
+    </span>
+  </label>
+</template>
+
+<script>
+import { ColorThemeMixin } from '../../../services/ColorThemePlugin'
+
+export default {
+  name: 'va-radio',
+  mixins: [ ColorThemeMixin ],
+  props: {
+    value: {
+    },
+    option: {
+      type: [Object, String, Number, Boolean],
+    },
+    name: {
+      type: [ String, Number ],
```

Comment:
This has spaces inside, the line above does not. Isn't something that linter should be taking care of?